libs/SalesforceSDKCore/SalesforceSDKCore/NativeLoginManagerTests.swift

```diff
@@ -108,4 +136,36 @@ final class NativeLoginManagerTests: XCTestCase {
         
         return user
     }
+    
+    /// A convenience for Native Login Manager's login method that discards errors.
```

Comment:
When I set up the login method to throw, I didn't want to introduce too much boilerplate error handling code to the tests which will fail on a nil result just as easily.  So, I was hoping this would improve the readability of the tests.

Reply:
I think in that scenario you could add `throws` to the test's signature and then any error would become a test failure automatically

Reply:
That really does make sense.  I'll have to give it a try, possibly for fun.  I'm also trying a version that doesn't throw 🙂